commons/connector/altoida/AltoidaPath.avsc

```diff
@@ -0,0 +1,13 @@
+{
+  "namespace": "radarcns.connector.altoida",
+  "type": "record",
+  "name": "AltoidaPath",
+  "doc": "Data of 3-axis distance travel in meters (m) to find?? or place?? an object.",
+  "fields": [
+    {"name": "time", "type": "double", "doc": "Timestamp in unix time acquired locally from the device."},
+    {"name": "timeRecieved", "type": "double", "doc": "Timestamp in unix time received from Altoida API."},
+    {"name": "x", "type": "float", "doc": "Distance travelled along the x axis (m)."},
```

Comment:
I don't think it's a travel distance, but rather just a distance to a fixed point. If we can figure out what that point is, like the initial point in the path, that would be good to document. But we may not know.